tests/core/test_search.py

```diff
@@ -66,7 +61,7 @@ def db_setup(test_db):
             True,
             None,
             None,
-            SortOrder.ASCENDING,
+            "asc",
```

Comment:
I don't understand why we are now using magic strings and not constants?

Reply:
Ah ok penny dropped - this is test data - and we want to emulate the frontend :)